### DISCUSSION TITLE
Possible Improvements to Participatory Mapping & Critical Cartography

### DISCUSSION THREAD

**Discussion by ghost** [albertkun/21S-ASIAAM-191A#65] [Past Thinking Caps]
Participatory mapping and critical cartography go hand in hand, meaning that I believe community-driven data is essential to impactful maps regarding critical topics in civil society. One way to foster this sector of cartography would be to increase and improve open source mapping platforms in accessibility and ease of use. Many people who have stories, opinions, experiences, etc to share may not see mapping as a way to do so because they do not have access to an inclusive platform or the skillset/education to use the available software. While I think it is amazing what we have been able to conquer in terms of addressing critical data and learning various forms of GIS and coding these past 2 weeks, the reality is that many people and communities do not get the guidance we have access to. I think that improving participatory mapping could mean putting straightforward open-source mapping platforms out for public use, maybe on computers at city libraries or for easy download on mobile devices. Libraries could even offer and advertise free workshops in tech-related fields so as to spread awareness that data and mapping can be inclusive. The problem isn't that revealing and critical data doesn't exist out in the world but that this data is hidden behind the misconception that mapping is only for the skilled or educated. As with anything, there are drawbacks to my above suggestions. Many people do not have easy access to libraries or do not own a phone or tech to access platforms/software. There are also many ways that corporations/the government overshadow even some of the most jarring critical data and maps. All in all, I think there are likely many more ways to improve participatory mapping and critical cartography but the main focus needs to be allowing people/communities to reclaim their voices and stories.

**Comment by Wny-Duong** [comment]
I've actually attended workshops at UCLA and back home offered by libraries specifically to provide people with digital skills such as programming languages, machine learning, data science, etc. The biggest issue in regards to that kind of stuff is actually encouraging good turnout. The most popular topics in tech such as data science, Python workshops (for some reason), machine learning will usually get immense amounts of turnout on their own, while it's stuff like GIS, or R and Tableau (more niche, specific tools), and the more obscure, less fashionable trends in tech that don't get a good turnout despite the workshops being offered. With stuff like workshops, you normally have to take things one step further and find out how you can make possible attendees care about the technology and its use cases. GIS and its relevant code are primarily restricted to the creation and dashboards, are mostly seen in areas regarding social issues, crimes, and humanitarian interests (at least, that's what the class has focused on and the current extent of my understanding of GIS's utility), which aren't seen in nearly the same glamorous light as more popular technology trends when applied to larger corporate or other revenue-focused interests. It's better, in my mind, to make GIS more popular as a tool for people with even the bare minimum of interest in topics like civil society and community development, as they'd be more receptive towards picking up GIS-related tools and doing something with them then the average person visiting a randomly-offered workshop.

**Comment by lvalles10** [comment]
Completely agree with everything stated above! It's interesting to see the level of nuance that's required for an intentional contribution to participatory mapping and critical cartography. In addition to more publicly available resources in libraries and other open source spaces, I think it's important to implement a participatory mapping and critical cartography education at and early age. It would serve elementary and middle schools students well to have a critical look at the spaces and places they call home. Perhaps an advent to improving participatory mapping and critical cartography could be to implement curriculum that emphasizes these two practices at an early age; not only would this better prepare future generations with the tools needed to best define the spaces that surround them, but it would also propagate a search for ethically sourced data in earnest. As with many concepts and critical practices, I believe early education is a way to improve the prominence of participatory mapping and critical cartography.

**Comment by ninalong** [comment]
Something interesting that I've noticed in a handful examples is that open-source and easy/accessible to use seem to be mutually exclusive at times. For example I've found that ArcGIS online is much more intuitive (and has many more user guides) than QGIS, the open source GIS software. SPSS is also easier to use than R, the open source stats software. I wonder how programmers can offer free platforms available to all without sacrificing quality.
